bool downsample
bool post_process
---
sensor_msgs/Image img